### PR TITLE
upstream merge 21/8

### LINKED ISSUE #1536: Flatpacker 1001 is fucked
## Description
Flatpacker spits out a base flatpack instead of the requested flatpack, and eats mats

**Reproduction**
Get flatpacker. Fill with mats. Give board. Print. Weep.

**Additional context**
Happened after upstream merge

### PR DESCRIPTION
**Pull request by deltanedas**
## Notable inclusions
- material price rebalance, resulted in having to price hike all the ships
- Material reclaimer removed, recycler emagged gibbing is still disabled though.
- cherry picked flatpack fix from **3 months ago**, fixes #1536

## Excluded
- Ore yield nerf (upstream #30920), need to discuss mining balance in context with everything. so its still 5 bars per ore
- Blacklisting sec from being pacifists or blind, for that 1 guy
- Energy shotgun commented out from hos locker and objective until we know what to do with it

## Technical Details
- added bluespace ore material to make tests not fail on arbitrage. there is no sprite for it and no rocks that yield it, so it's still unused.

**Changelog**
:cl:
- add: Merged upstream, except for mining rebalance and energy shotgun.
- tweak: Every shuttle's price increased due to the rising cost of materials.
- fix: Fixed the Flatpacker being broken.

### CODE REVIEW

**Comment by Axionyxx** [comment]
> DeltaV/Maps/salvage_modified.yml removed since prefixing the ids with DV didnt actually make it replace the originals. now it replaces them properly and adds deltav comments so its easier to keep up to date, instead of the entire file being conflict

Did you test this in game? The actual salvage map file is disabled

**Comment by deltanedas** [comment]
> Did you test this in game? The actual salvage map file is disabled

disabled with what, ive seen default wrecks quite a bit

**Comment by Axionyxx** [comment]
> > Did you test this in game? The actual salvage map file is disabled
> 
> disabled with what, ive seen default wrecks quite a bit

https://github.com/DeltaV-Station/Delta-v/blob/master/Resources/IgnoredPrototypes/ignoredPrototypes.yml

**Comment by deltanedas** [comment]
ah its good to unignore then so small asteroids exist, they never had a port or anything so they just never got rolled, possibly commenting engineering chunk

**Comment by Axionyxx** [comment]
You do you, just making you aware that pushing it like this won't have the intended effect

**Comment by IamVelcroboy** [comment]
> ah its good to unignore then so small asteroids exist, they never had a port or anything so they just never got rolled, possibly commenting engineering chunk

We decided we didn't want the small ones a while ago. Just weren't worth it.

**Comment by IamVelcroboy** [comment]
Also, we've made significant changes to our versions of the wrecks. So, we don't want to revert back to the upstream ones.

**Review by IamVelcroboy — CHANGES_REQUESTED** [review]
Don't do this. This was done for a reason.

**Comment by deltanedas** [comment]
> Also, we've made significant changes to our versions of the wrecks. So, we don't want to revert back to the upstream ones.

dw its still using our large ones (except the engineering chunk which doesnt have one)

if the small ones arent wanted then ig its fine to ignore the prototype, and added a comment linking to the ignored prototypes for anyone looking at it in the future

**Comment by Axionyxx** [comment]
> > Also, we've made significant changes to our versions of the wrecks. So, we don't want to revert back to the upstream ones.
> 
> dw its still using our large ones (except the engineering chunk which doesnt have one)
> 
> if the small ones arent wanted then ig its fine to ignore the prototype

Whats the harm in keeping it as is? It avoids merge conflicts during upstream merges, and new wrecks aren't automatically available to players

**Comment by Axionyxx** [comment]
Don't remove the DV prefixes, it needs them. Ignored prototypes are abstracted, not removed.

**Comment by rosieposieeee** [comment]
personally i think the energy shotgun should be kept on hos and their old laser gun should be given to warden (and kept as a steal target)

it looks like a fun weapon :)

**Comment by IamVelcroboy** [comment]
> personally i think the energy shotgun should be kept on hos and their old laser gun should be given to warden (and kept as a steal target)
> 
> it looks like a fun weapon :)

In Discord, we discussed making it a rng spawn and have both weapons satisfy the objective. Can be done easily after the merge.

File: Resources/Prototypes/DeltaV/Maps/salvage_modified.yml [hunk 1]
```diff
@@ -1,105 +1,112 @@
...
   id: DVMedium1
   mapPath: /Maps/Salvage/DeltaV/DV-medium-01.yml
-  size: medium
+  sizeString: salvage-map-wreck-size-medium
 
 - type: salvageMap
   id: DVMediumVault1
-  mapPath: /Maps/Salvage/DeltaV/DV-med-vault-01.yml
-  size: medium
+  mapPath: /Maps/Salvage/DeltaV/DV-medium-01.yml
```
Comment (IamVelcroboy) [inline Resources/Prototypes/DeltaV/Maps/salvage_modified.yml]: This one should still say `med-vault` or it will duplicate the one above it

Comment (deltanedas) [reply Resources/Prototypes/DeltaV/Maps/salvage_modified.yml]: typo ops

**Comment by deltanedas** [comment]
either solution will be trivial since the spawn is commented out thats all

**Comment by Avalon-Proto** [comment]
> > personally i think the energy shotgun should be kept on hos and their old laser gun should be given to warden (and kept as a steal target)
> > it looks like a fun weapon :)
> 
> In Discord, we discussed making it a rng spawn and have both weapons satisfy the objective. Can be done easily after the merge.

I do think we should always have both guns available though, I would personally vote in giving the X-01 to ward, and giving the shotgun to HoS. That was both are in play, as both are unique, and powerful, weapons

**Pull request merged**